_data/finance.yml

```diff
@@ -406,7 +406,7 @@ websites:
 
     - name: You Need A Budget (YNAB)
       url: https://www.youneedabudget.com/
-      twitter: ynab
-      facebook: iYNAB
       img: ynab.png
-      tfa: No
+      tfa: Yes
+      software: Yes
+      doc: https://docs.youneedabudget.com/article/1364-two-step-verification
```

Comment:
```suggestion
      doc: https://docs.youneedabudget.com/article/1364--
```

Reply:
What's the purpose of removing the article slug?

I chose the address with slug as it's what is suggested via YNAB's docs search function:
![image](https://user-images.githubusercontent.com/10128574/65413617-baa4ed00-dde9-11e9-8f23-0cd7a4c2e1ee.png)

Reply:
We have removed the slug from every HelpCenter and HelpScout addresses for size reduction reasons and because "it works". Also, we had the assurance from HelpScout that they were working on removing the need for a slug at all in the coming months.